.eslintrc.json

```diff
@@ -5,13 +5,14 @@
         "es2021": true
     },
     "extends": [
-        "react-app",
-        "eslint:recommended", 
-        "airbnb", 
-        "airbnb/hooks", 
-        "airbnb-typescript", 
+        "eslint:recommended",
+        "airbnb-base",
```

Comment:
Why are we replacing `airbnb` with `airbnb-base`? The former includes support for react [docs](https://www.npmjs.com/package/eslint-config-airbnb)

Reply:
will test out this change